sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/InvocationVisitor.cs

```diff
@@ -130,7 +137,9 @@ private ProgramState HandleStringNullOrEmptyCheckMethod()
             {
                 newProgramState= newProgramState.SetConstraint(refEquals, BoolConstraint.True);

```

Comment:
please give better names for `arg1` and `arg2` in `HandleStringNullOrEmptyCheckMethod` as well